pkgs/tools/networking/burpsuite/default.nix

```diff
@@ -22,20 +22,20 @@ let
     hash = product.hash;
   };
 
-  pname = "burpsuite";
+  name = "burpsuite-${version}";
```

Comment:
Why does this version bump commit revert the pname/version change from an earlier commit in this PR?